commons/src/main/java/com/powsybl/commons/config/MapModuleConfig.java

```diff
@@ -64,41 +63,70 @@ private static String substitureEnvVar(String str) {
         return str.replace("$HOME", System.getProperty("user.home"));
     }
 
+    private static PowsyblException createPropertyNotSetException(String name) {
+        return new PowsyblException("Property " + name + " is not set");
+    }
+
+    private static PowsyblException createUnexpectedPropertyTypeException(Object value) {
```

Comment:
Add the property name and the expected type to give more details in the error message

Reply:
Done